datumaro/components/operations.py

```diff
@@ -335,6 +336,9 @@ def _merge_label_categories(self, sources):
     def _merge_point_categories(self, sources, label_cat):
         dst_point_cat = PointsCategories()
 
+        if all(AnnotationType.points not in cats for cats in sources):
+            return None
```

Comment:
What's the reason for adding this check? Seems like the general case implementation will produce the same result.

Reply:
Removed. There was some failing case, but I can't reproduce it now.